doc/xds-test-descriptions.md

```diff
@@ -20,8 +20,12 @@ Server should accept these arguments:
 In addition, when handling requests, if the initial request metadata contains the `rpc-behavior` key, it should modify its handling of the request as follows:
 
  - If the value matches `sleep-<int>`, the server should wait the specified number of seconds before responding.
- - If the value matches `fail=<status code as int>`, the server should respond with the specified status code.
- - A value can have a prefix `hostname=<string>` followed by a space. In that case, the rest of the value should only be applied if the specified hostname matches the server's hostname. 
+ - If the value matches `keep-open`, the server should never respond to the request.
+ - If the value matches `success-on-retry-attempt-<int>`, the server should force an OK response if the value of the `grpc-previous-rpc-attempts` metadata field is equal to the specified number.
```

Comment:
I assume that this is supposed to override the behavior of the `error-code` option, and that's what I was trying to communicate with "forces an OK response". Maybe I should just say that explicitly.

Reply:
I updated the wording. Is that better?

Reply:
I don't think it is "don't do `error-code-`" as much as "don't process any more behaviors." It is a terminal state, essentially. As written now it sounds like if an `error-code-` exists earlier in the list this will retroactively ignore it.

Does something like this work better?
 - If the value matches `sleep-<int>`, the server should wait the specified number of seconds before resuming behavior matching and RPC processing.
 - If the value matches `keep-open`, the server should never respond to the request and behavior matching ends.
 - If the value matches `error-code-<int>`, the server should respond with the specified status code and behavior matching ends.
 - If the value matches `success-on-retry-attempt-<int>` and the value of the `grpc-previous-rpc-attempts` metadata field is equal to the specified number, the normal RPC processing should resume and behavior matching ends.

I think there's some other ways this could be phrased, but I was trying to sort of match the existing style. Some rearrangements could reduce the verbosity, but we've already spent more time on wording this than we were hoping for.

Reply:
I updated it to use your wording.